README.md

```diff
@@ -34,4 +34,5 @@ unpriviledged user, the space pilot hidraw device must have the appropriate
 permissions. The best way to do that, is to add a new file `3dxdisp.rules` to
 `/etc/udev/rules.d/` with the following contents:
 
-    KERNEL=="hidraw[0-9]*", ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c625", MODE="0666"
+    KERNEL=="hidraw[0-9]*", ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c625", MODE="0666", GROUP="<user_gid>", SYMLINK+="spacenav"
```

Comment:
I like the idea of setting up a custom symlink, but I have two suggestions:
1. since this is specifically for the space pilot, I think the link should be called something like spacepilot instead.
2. the group, with the extra step of substituting one, doesn't seem to do anything here, since the mode is 0666, so I'd rather just drop it.

Reply:
right, but 0666 is never good choice. 

mode 0660 and group=1000 is what i use.

in some default setup (on distribution packages) group can be input/console/plugdev/<some new group>

Reply:
Right, but this is an example, on what is probably a single-user machine anyway. Let's not complicate matters needlesly. Distributions, if they pick this program up for some reason, should use whatever is more appropriate.